ui/imports/utils/Constants.qml

```diff
@@ -221,4 +221,6 @@ QtObject {
         //% "Continuing will require a transaction to connect the username with your current chat key."
         "connected-different-key": qsTrId("ens-username-connected-with-different-key"),
     }
+
+    readonly property bool isCppApp: Qt.application.name === "Status Desktop C++"
```

Comment:
please, you can provide a singleton from C++ to QML, but checking the name of app is too unstable.

Reply:
as discussed exposed via context property for now. :)